### PR TITLE
Issue 2 select category - Mike Glenn

### PR DESCRIPTION
**Pull request by mikejglenn**
Animation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,7 +95,79 @@
...
+                    <option value="27">Animals</option>
+                    <option value="28">Vehicles</option>
+                    <option value="29">Entertainment: Comics</option>
+                    <option value="30">Science: Gadgets</option>
+                    <option value="31">
+                      Entertainment: Japanese Anime &amp; Manga
+                    </option>
+                    <option value="32">
+                      Entertainment: Cartoon &amp; Animations
+                    </option>
```
Comment (shawnkost) [inline index.html]: Are you able to extract this information from the API rather than hard coding yourself? What if the API adds more category options?

**Review by shawnkost — APPROVED** [review]
Nicely done!

File: index.html (hunk 1 above)
Comment (mikejglenn) [reply index.html]: Yes they provide a way to do this:
Category Lookup: Returns the entire list of categories and ids in the database.
https://opentdb.com/api_category.php

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**